### DISCUSSION TITLE
Welcome to uvg266 Discussions!

### DISCUSSION THREAD

**Discussion by fador** [ultravideo/uvg266#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

You can also join our [Discord](https://discord.gg/fZpub7BPUA) for more real-time discussions.

**Comment by maa7** [comment]
Is the project dead? (two years without new releases)

**Reply by fador** [reply]
We are still working on uvg266 but with only two people more or less actively developing, it's not that fast. We are also trying to get research output at the same time 😅
